### DISCUSSION TITLE
Availability of BleachBit in PPA

### DISCUSSION THREAD

**Discussion by mryujitanaka** [bleachbit/bleachbit#1078] [Ideas]
Is there a possibility that one day we have a version of BleachBit in PPA?

**Comment by az0** [comment]
I appreciate the convenience, but the problem with automatic packages updates is explained here https://github.com/bleachbit/bleachbit/issues/342#issuecomment-483932944

**Reply by mryujitanaka** [reply]
It is a pity. But I understand.
